docs/release_notes/ibexa_dxp_v5.0_deprecations.md

```diff
@@ -515,6 +516,104 @@ Support for facet search has been dropped, use the `Aggregation` API instead.
 | `\Ibexa\Contracts\Workflow\Service\WorkflowServiceInterface::loadWorkflowMetadataOriginatedByUser` | Removed |
 | `\Ibexa\Contracts\Workflow\Service\WorkflowServiceInterface::loadAllWorkflowMetadata` | Removed |
 
+## PHP method parameters
+
+The `ValueObject` argument was replaced by `object` in a number of interfaces in `core` and `migrations` package.
+In `core`, this change improves extensibility by enabling the use of custom object types to be interpreted by `PermissionResolver`.
+In `migrations`, it makes it easier to integrate custom data types, especially when using `AbstractStepFactory`.
+
+### ibexa/core
+
+| PHP Interface                                              | Methods                                                                |
+|:------------------------------------------------------|:------------------------------------------------------------------------|
+| `src/contracts/Repository/PermissionResolver.php` | `canUser`, `lookupLimitations` |
```

Comment:
I would have use FQCN:

```suggestion
| `\Ibexa\Contracts\Core\Repository\PermissionResolver` | `canUser`, `lookupLimitations` |
```

Maybe even some links to the PHP API Ref when in the Contracts namespace:

```suggestion
| `\Ibexa\Contracts\Core\Repository\PermissionResolver` | [`canUser`](/api/php_api/php_api_reference/classes/Ibexa-Contracts-Core-Repository-PermissionResolver.html#method_canUser), [`lookupLimitations`](https://doc.ibexa.co/en/5.0/api/php_api/php_api_reference/classes/Ibexa-Contracts-Core-Repository-PermissionResolver.html#method_lookupLimitations) |
```